_layouts/indicator.html

```diff
@@ -171,5 +173,29 @@ <h4>{{ t.indicator.sub_categories }}</h4>
   </div>
 </div>
 
-{% include footer.html %}
+<style type="text/css">
```

Comment:
Could this be added in the [Sass file](https://github.com/open-sdg/open-sdg/blob/master/assets/css/default.scss)?